cvxpy/lin_ops/canon_backend.py

```diff
@@ -1103,6 +1090,319 @@ def _to_dense(x):
         return res
 
 
+class StackedSlicesBackend(PythonCanonBackend):
+
+    @staticmethod
+    def reshape_constant_data(constant_data: Any, lin_op_shape: tuple[int, int]) \
+            -> dict[int, sp.csc_matrix]:
+        return {k: StackedSlicesBackend._reshape_single_constant_tensor(v, lin_op_shape)
+                for k, v in constant_data.items()}
+
+    @staticmethod
+    def _reshape_single_constant_tensor(v: sp.csr_matrix, lin_op_shape: tuple[int, int]) \
+            -> sp.csc_matrix:
+        p = np.prod(v.shape) // np.prod(lin_op_shape)
+        old_shape = (v.shape[0] // p, v.shape[1])
+
+        coo = v.tocoo()
+        data, stacked_rows, cols = coo.data, coo.row, coo.col
+        slice, rows = np.divmod(stacked_rows, old_shape[0])
+
+        element_position = cols * old_shape[0] + rows
+        new_cols, new_rows = np.divmod(element_position, lin_op_shape[0])
+        new_rows = slice * lin_op_shape[0] + new_rows
+
+        new_stacked_shape = (p * lin_op_shape[0], lin_op_shape[1])
+        return sp.csc_matrix((data, (new_rows, new_cols)), shape=new_stacked_shape)
+
+    def get_empty_view(self) -> TensorView:
+        return StackedSlicesTensorView.get_empty_view(self.param_size_plus_one, self.id_to_col,
+                                                      self.param_to_size, self.param_to_col,
+                                                      self.var_length)
+
+    def mul(self, lin: LinOp, view: StackedSlicesTensorView) -> StackedSlicesTensorView:
+        """
+        Multiply view with constant data from the left.
+
+        """
+        lhs, is_param_free_lhs = self.get_constant_data(lin.data, view, column=False)
+        if is_param_free_lhs:
+            reps = view.rows // lhs.shape[-1]
+            stacked_lhs = (sp.kron(sp.eye(reps, format="csc"), lhs))
+
+            def func(x, _p):
+                return stacked_lhs.tocsc() @ x
```

Comment:
It seems like there is no need to implement if p > 1 here,
I added a breakpoint for all test cases and it hasn't stopped.